SoftLayer/fixtures/SoftLayer_Account.py

```diff
@@ -1100,3 +1100,110 @@
         "smtpAccess": "1"
     }
 ]
+
+getActiveAccountLicenses = [{
+    "accountId": 123456,
+    "capacity": "4",
+    "key": "M02A5-6CJ8L-J8R9H-000R0-CDR70",
```

Comment:
```suggestion
    "key": "M0CK5-TEST5-KEY00-000R0-CDR70",
```
Also, I think just an item on the list would be enough.

Reply:
is better a randomize this key